sample.js

```diff
@@ -0,0 +1,5 @@
+const obj ={
+    name:'ahmad',
```

Comment:
you can write a full name here instead of first name . please make it as "Sk.Ahmad Pasha" at name property in the object

Reply:
okay will do that change